compiler-rt/test/asan/TestCases/Windows/aligned_mallocs.cpp

```diff
@@ -1,5 +1,5 @@
 // RUN: %clang_cl_asan %Od %s %Fe%t
-// RUN: %run %t
+// RUN: %run %t | FileCheck %s
```

Comment:
Since we expect the program execution to fail with an asan error, we need the "not" prefix. (Also since the asan diagnostic will print to stderr, we should probably redirect it):

```
// RUN: not %run %t 2>&1 | FileCheck %s
```

Additionally, the test does not currently work when statically linking against the crt. We need to update asan_malloc_windows_thunk.cpp with the new functions as well.